Cargo.toml

```diff
@@ -6,7 +6,7 @@ members = [
     "wgpu-types",
 ]
 
-#[patch."https://github.com/gfx-rs/gfx"]
+[patch."https://github.com/gfx-rs/gfx"]
```

Comment:
I think this slipped in

Reply:
that was intentional, actually. Having a patch section without contents isn't doing anything, but enabling/disabling overrides is easier

Reply:
Oh I didn't realize that was valid, nice